Lombiq.DataTables.Tests.UI/Models/TableDrawState.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace Lombiq.DataTables.Tests.UI.Models;
 
+[Obsolete("This class no longer works after the latest DataTables update, and will be removed in the future.")]
```

Comment:
If it doesn't work, why not remove it? Due to the DT upgrade, this is a breaking PR anyway.

Reply:
To leave up information for the unlikely case that someone actually used it.